host_key_rotation.py

```diff
@@ -99,124 +98,10 @@ def rotate_meetings():
 
 **Abstract:** {{abstract}}
 
-Enjoy the talk,  
+Enjoy the talk,
```

Comment:
Please revert, this is markdown's newline

```suggestion
Enjoy the talk,  
```

Reply:
I am not aware of changing this, and I can't see the characters. Perhaps my IDE (Atom) did this. Just add a new line?

Reply:
Your IDE likely automatically deletes trailing whitespace, which isn't correct in markdown (except here we're having markdown in Python).

Reply:
Good catch, in that case. I've added them back explicitly, let's see